Samples/ExtendedControlAndMetadata/EyeGazeAndBackgroundSegmentation/EyeGazeAndBackgroundSegmentation/App.xaml.cs

```diff
@@ -0,0 +1,92 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using Windows.ApplicationModel;
+using Windows.ApplicationModel.Activation;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Navigation;
+
+namespace EyeGazeAndBackgroundSegmentation
+{
+    /// <summary>
+    /// Provides application-specific behavior to supplement the default Application class.
+    /// </summary>
+    sealed partial class App : Application

```

Comment:
>Application [](http://example.com/codeflow?start=31&length=11)

We are mixing c++/C#/C++ cx is it possible to do it all in the same language?

Reply:
- I could port the app code to C++/WinRT, but it is a bit more work.. C# is used only for the UI and camera setup portion here.
- we should avoid using C+/CLI from now on.  I don't mix C++/CLI  with other language in these samples, I kept the exiting one and made sure it worked / fixed issue.

Reply:
I don't know how others feel about this, letting an app code the UI portion using C# is way more efficient in term of code and maintenance, and C++/WinRT is used to bridge any lower level / pointer-syntax related  / architecture-specific needs

Reply:
I am ok with  c++/c# if possible thought the cx stuff should be rewritten it could be addressed in another itteration though

Reply:
I agree